FreeRTOS/FreeRTOS-Plus-TCP
# Remove error message from the unit test build logs

Description
-----------
This PR suppresses an error message displayed while building the unit tests, which happens due to improper parsing of the header files by `unifdefall` . Also the coroutines are disabled in the FreeRTOS config files used for building the unit tests and other tools.

Test Steps
-----------
Build the unit tests.

Checklist:
----------

Related Issue
-----------
Error message in the unit test build.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
